ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -258,14 +259,12 @@ private static void ConfigureApp([SuppressMessage("ReSharper", "SuggestBaseTypeF
 		app.MapOpenApi("/swagger/{documentName}/swagger.json");
 
 		// Add support for swagger UI, this should be after swagger, obviously
-		app.UseSwaggerUI(
-			static options => {
-				options.DisplayRequestDuration();
-				options.EnableDeepLinking();
-				options.EnableTryItOutByDefault();
-				options.ShowCommonExtensions();
-				options.ShowExtensions();
-				options.SwaggerEndpoint($"{SharedInfo.ASF}/swagger.json", $"{SharedInfo.ASF} API");
+		app.MapScalarApiReference(
+			"/swagger", static options => {
+				options.DefaultFonts = false;
+				options.OpenApiRoutePattern = $"/swagger/{SharedInfo.ASF}/swagger.json";
+				options.Theme = ScalarTheme.Kepler;
```

Comment:
I kinda like [this](https://windows98.apidocumentation.com/version-0/reference#tag/pet) theme 😄

Reply:
We consider only built-in themes regardless :stuck_out_tongue: